src/test/java/com/thomashan/coup/turn/state/WaitingMainActionStateTestCase.java

```diff
@@ -0,0 +1,7 @@
+package com.thomashan.coup.turn.state;
+
+public class WaitingMainActionStateTestCase extends TurnStateTestCase {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This class name ends with Test but contains no test cases](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)